config/routes.rb

```diff
@@ -3,5 +3,9 @@
   devise_for :users
   root to: 'books#index'
   resources :books
-  resources :users, only: %i(index show)
+  resources :users, only: %i[index show] do
+    resources :friendships, only: %i[create destroy]
+    resources :following, only: %i[index], module: :users
```

Comment:
複数のリソースを扱うので複数形になっていたほうがよいですね。

```suggestion
    resources :followings, only: %i[index], module: :users
```

Reply:
単数・複数の区別がない単語の認識で複数形のsをあえてつけてないのですが、修正した方がよろしいでしょうか。